build/yamls/flow-visibility/base/kustomize-config.yml

```diff
@@ -0,0 +1,5 @@
+# This file defines the resource and the field that Kustomize need to look up for
+# the value of var `CH_CONF`
```

Comment:
The comment seems a bit backwards to me. These are the "extra" (non built-in) paths where kustomize needs to look for variable substitutions if needed. The value of the variable comes from the ConfigMap declaration.